synapse/lib/certdir.py

```diff
@@ -319,19 +319,6 @@ def getHostCaPath(self, name):
 
         return capath
 
-    def getHostCaPath(self, name):
```

Comment:
duplicated twice https://github.com/vertexproject/synapse/blob/master/synapse/lib/certdir.py#L309 and https://github.com/vertexproject/synapse/blob/master/synapse/lib/certdir.py#L322